docs/en/operation/Scalability-Tuning.md

```diff
@@ -139,6 +141,15 @@ ALLUXIO_MASTER_JAVA_OPTS+=" -Xms256g -Xmx256g "
 When setting the heap size, ensure that there is enough memory allocated for off heap storage.
 For example, spawning `4000` threads with a default thread stack size of `1 MB` requires at least
 `4 GB` of off-heap space available.
+* Network buffers are often allocated from a pool of direct memory in Java. 
+The configuration controlling the maximum size of direct memory allocated defaults to the xmx setting, 
+which can leave very little space for the other critical processes in the system. 
+We recommend setting it to 10GB for both Alluxio Master and Alluxio Workers in a typical deployment, and only increase it
```

Comment:
I thought big direct memory is only needed for workers as they do large data IOs. Is that the case for the master too?

Reply:
master can do large data /network IO too? eg pinlist?